### DISCUSSION TITLE
Make the quality gateway pass

### DISCUSSION THREAD

**Discussion by ruanwenjun** [apache/dolphinscheduler#11065] [Q&A] [answered]
I find the [quality gateway](https://sonarcloud.io/project/overview?id=apache-dolphinscheduler) is failed, do we have any way to make this pass? cc @SbloodyS @zhongjiajie 
<img width="910" alt="image" src="https://user-images.githubusercontent.com/22415594/179885638-16058f57-0293-4d9c-8c5a-0c1f7225821e.png">

**Comment by zhongjiajie** [comment, marked as answer]
It is seem is unit test coverage failed our quality gate, https://sonarcloud.io/project/overview?id=apache-dolphinscheduler

**Reply by zhongjiajie** [reply]
And the reason why coverage too less is in https://github.com/apache/dolphinscheduler/pull/11084 contributor merge upstream/dev to current PR (you can search by keyword `Merge branch 'dev'`). Our sonar track the code change, and all merging code is considered to code change. due to our coverage failed

**Reply by zhongjiajie** [reply]
Maybe we should change sonar strategy from track change to check the whole unit test coverage, maybe it can solve this problem

**Reply by ruanwenjun** [reply]
OK, so if we increase the UT coverage, the check can pass.
